src/Raven.Server/Integrations/PostgreSQL/PgQuery.cs

```diff
@@ -51,7 +51,14 @@ public static PgQuery CreateInstance(string queryText, int[] parametersDataTypes
                     return powerBiQuery;
                 }
 
-                return new HardcodedQuery(queryText, parametersDataTypes);
+                if (HardcodedQuery.TryParse(queryText, parametersDataTypes, out var hardcodedQuery))
+                    return hardcodedQuery;
+
+                throw new PgErrorException(
+                    PgErrorCodes.StatementTooComplex,
+                    "Unhandled query (Are you using ; in your query? " +
+                    "That is likely causing Power BI to split the query and results in partial queries): " +
```

Comment:
I know that Power BI will put it in a singe line anyway but other clients might format it better. Can we put `Environment.NewLine` before we add the query text?